content/foundation/bylaws/index.md

```diff
@@ -0,0 +1,273 @@
++++
+title = "Bevy Foundation Bylaws"
+template = "news-page.html"
+date = 2024-02-03
+[extra]
+author = "Bevy Foundation Board of Directors"
+header_message = "Foundation Bylaws"
++++
+
+## Article 1. Offices
+
+The principal office of Bevy Foundation (the “Corporation") shall be located at its principal place of business or such other place as the Board of Directors (“Board”) may designate. The Corporation may have other such offices, either within or without the State of Washington, as the Board may designate or as the business of the Corporation may require from time to time.
```

Comment:
```suggestion
The principal office of Bevy Foundation (the “Corporation”) shall be located at its principal place of business or such other place as the Board of Directors (“Board”) may designate. The Corporation may have other such offices, either within or without the State of Washington, as the Board may designate or as the business of the Corporation may require from time to time.
```

Sorry, I suggested a non-matching double quote character and you clicked the button before I managed to edit in the right one.